### PR TITLE
feat: add optional color for slide-toggle when unactivated

### PR DESCRIPTION
**Pull request by willdp**
add optional color for slide0toggle when it is unactivated, also created
storybook for each case

### CODE REVIEW

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            cod: "2",
+            checked: true,
+            title: "both colors",
+            color: "#10c0c6",
+            uncheckedColor: "#456888"
+        },
+        {
+        cod: "3",
+        checked: false,
+        title: "UNACTIVATE COLOR",
```
Comment (DenisOkawa) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Shouldn't the title be 'Inactive'?

File: libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { FormControl } from "@angular/forms";
 export class SlideToggleComponent implements OnInit {
   checked = true;
   colorFormControl = new FormControl("#08B25A");
+  uncheckedColorFormControl = new FormControl("#000000")
```
Comment (oliveira-gust) [inline libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts]: missing ;

Comment (oliveira-gust) [reply libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts]: Also, please add the access modifier for these properties

File: libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.html [hunk 3]
```diff
@@ -24,6 +24,17 @@
...
       </mat-form-field>
     </mat-list-item>
+    <mat-list-item role="listitem" class="color-config-container">
+      <mat-form-field class="full-width">
+        <input
+          matInput
+          type="color"
+          placeholder="Unchecked Color"
+          [formControl]="uncheckedColorFormControl"
+        />
```
Comment (oliveira-gust) [inline libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.html]: `/>` could be moved to the previous line

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle-group.component.html [hunk 4]
```diff
@@ -7,11 +7,12 @@
...
+          [uncheckedColor]="toggle?.uncheckedColor"
           [title]="toggle?.title"
           (toggleChange)="toggleChange(toggle?.cod, $event)"
           ngDefaultControl
         ></gor-slide-toggle>
       </div>
     </ng-container>
   </div>
-</ng-container>
+</ng-container>
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle-group.component.html]: Missing newline at end of file

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.component.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
   <label class="switch">
     <input type="checkbox" [checked]="checked" (change)="changeChecked()" />
-    <span class="slider round" [style.backgroundColor]="checked ? color : ''">
+    <span class="slider round" [style.backgroundColor]="checked ? color : uncheckedColor">
       <span class="slider-before" [ngClass]="{ active: checked }"></span>
     </span>
   </label>
   <span class="title">{{ title }}</span>
-</div>
+</div>
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.component.html]: Missing newline at end of file

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.component.ts [hunk 6]
```diff
@@ -22,6 +22,12 @@ export class SlideToggleComponent {
   @Input() public color = '';
 
   /**
+   * The toggle color when unchecked.
+   */
+  @Input() public uncheckedColor = '';
+
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.component.ts]: Please remove extra empty line

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts [hunk 7]
```diff
@@ -22,8 +22,15 @@ export interface ToggleData {
 
 
   /**
-   * The custom color for the toggle.
+   * The custom color for the toggle when activated.
    * @Optional
    */
   color?: string;
+
+
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts]: Please remove extra empty line

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts [hunk 8]
```diff
@@ -22,8 +22,15 @@ export interface ToggleData {
...
+    /**
+   * The custom color for the toggle when unactivated.
+   * @Optional
+   */
+  uncheckedColor?: string;
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts]: Please include access modifier for all properties

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+storiesOf('slide-toggle', module)
+  .addDecorator(
+    moduleMetadata({
+        imports: [CommonModule],
+        declarations: [SlideToggleComponent, SlideToggleGroupComponent],
+ })
+)
+.addDecorator(withKnobs)
+.add(
+    'three different cases about slide-toggle',
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Summary should be descriptive of functionality being demonstrated - in this case, color configurations

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+import { CommonModule } from '@angular/common';
+import { SlideToggleComponent } from './slide-toggle.component';
+import { SlideToggleGroupComponent } from './slide-toggle-group.component';
+import { ToggleData } from './slide-toggle.model';
+import { text, withKnobs, boolean } from '@storybook/addon-knobs';
+
+storiesOf('slide-toggle', module)
+  .addDecorator(
+    moduleMetadata({
+        imports: [CommonModule],
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Check if this import is necessary

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 11]
```diff
@@ -0,0 +1,74 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Nice stories, keep up the good work

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+            title: "both colors",
+            color: "#10c0c6",
+            uncheckedColor: "#456888"
+        },
+        {
+        cod: "3",
+        checked: false,
+        title: "UNACTIVATE COLOR",
+        color: "",
+        uncheckedColor: "#456888"
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Check code identation

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts (hunk 1 above)
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Keep text capitalization consistent across titles

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+       <gor-slide-toggle
+       [id]="cod"
+       [checked]="checked"
+       [color]="color"
+       [uncheckedColor]="uncheckedColor"
+       [title]="title"
+       >
+        </gor-slide-toggle>
+        `
+    }));
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Missing newline at end of file

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts (hunk 1 above)
Comment (oliveira-gust) [reply libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Or maybe 'unchecked', to keep consistency with component property

File: .prettierignore [hunk 14]
```diff
@@ -4,3 +4,4 @@
 **/*.md
 **/*.scss
 **/*.html
+**/*.stories.ts
```
Comment (oliveira-gust) [inline .prettierignore]: Missing newline at end of file

File: libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts [hunk 15]
```diff
@@ -2,18 +2,18 @@ import { Component, OnInit } from "@angular/core";
...
-  titleFormControl = new FormControl("click me");
+  public checked = true;
+  public colorFormControl = new FormControl('#08B25A');
+  public uncheckedColorFormControl = new FormControl('#000000');
+  public disabled = false;
+  public titleFormControl = new FormControl('click me');
 
   constructor() {}
 
   ngOnInit() {}
```
Comment (oliveira-gust) [inline libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts]: Empty methods can be removed

File: libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts [hunk 16]
```diff
@@ -2,18 +2,18 @@ import { Component, OnInit } from "@angular/core";
...
 
 @Component({
-  selector: "gorilainvest-slide-toggle",
-  templateUrl: "./slide-toggle.component.html",
-  styleUrls: ["./slide-toggle.component.scss"]
+  selector: 'gorilainvest-slide-toggle',
+  templateUrl: './slide-toggle.component.html',
+  styleUrls: ['./slide-toggle.component.scss']
 })
 export class SlideToggleComponent implements OnInit {
```
Comment (oliveira-gust) [inline libs/shell/controls/src/lib/ui/slide-toggle/slide-toggle.component.ts]: OnInit interface can be removed

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+          cod: '2',
+          checked: true,
+          title: 'BOTH COLORS',
+          color: '#10c0c6',
+          uncheckedColor: '#456888'
+        },
+        {
+          cod: '3',
+          checked: false,
+          title: 'UNCHEKED COLOR',
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Please correct typo in title

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+          color: '',
+          uncheckedColor: '#456888'
+        }
+    ];
+
+      const labels = ['title'];
+
+      return {
+
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Remove extra empty line

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+      template: `
+          <gor-slide-toggle-group
+              [data]='${JSON.stringify(data)}'
+              [labels]='${JSON.stringify(labels)}'
+          >
+          </gor-slide-toggle-group>
+    `
```
Comment (oliveira-gust) [inline libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts]: Check identation

File: libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts (hunk 8 above)
Comment (guilhermejcgois) [reply libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts]: Interfaces define public contracts, so all members is public and no modifier is required

**Comment by guilhermejcgois** [comment]
Two more things before merge the PR:

1. Fix last commits author
2. Update branch

**Pull request closed without merging**